packages/domain-picker/src/components/domain-suggestion-item/suggestion-item-wrapper.tsx

```diff
@@ -20,7 +20,7 @@ type WrappingComponentProps = WrappingComponentAdditionalProps &
 
 const WrappingComponent = React.forwardRef< HTMLButtonElement, WrappingComponentProps >(
 	( { type, disabled, ...props }, ref ) => {
-		if ( type === 'button' ) {
+		if ( type === SUGGESTION_ITEM_TYPE_BUTTON ) {
```

Comment:
[nitpick/] This is scope creep since it doesn't have anything to do with folder restructuring.

Reply:
That's partially true, although the reason these changes are in this PR is because initially I had extracted these variables to a separate file during the folder restructure, and then I changed my mind and moved them back.

It's a small scope creek but probably we can keep it in, since the changes are not affecting the functionality of the component?

Reply:
Thanks for elaborating. Although we have to consider it as scope creep, we can keep it in since the changes are not affecting the functionality of the component